optimum/exporters/openvino/model_configs.py

```diff
@@ -2573,19 +2582,32 @@ def __init__(
         num_channels: int = DEFAULT_DUMMY_SHAPES["num_channels"],
         width: int = 336,
         height: int = 336,
+        crop_size=336,
         **kwargs,
     ):
         self.batch_size = batch_size
         self._embed_layer_realization = normalized_config.config.embd_layer["embedding_cls"]
-        self.image_dim_out = normalized_config.config.img_processor["image_dim_out"]
+        self.image_dim_out = (
+            normalized_config.config.img_processor.get(
+                "image_dim_out", normalized_config.config.img_processor.get("hidden_size")
+            )
+            if normalized_config.config.img_processor is not None
+            else 1152
+        )
         self.height = height
         self.width = width
+        if "image_embd_layer" in normalized_config.config.embd_layer:
+            self.crop_size = normalized_config.config.embd_layer.get("image_embd_layer", {}).get(
```

Comment:
this can be removed as you check above whether `image_embd_layer` is present already (so will never be set to this default value)
```suggestion
            self.crop_size = normalized_config.config.embd_layer.get("image_embd_layer").get(
```
in case this was added for cases where they key exist but the value set to None then it should be updated

Reply:
there is no such key in phi3-vision and this is needed to differentiate one model from another